recipe/meta.yaml

```diff
@@ -19,11 +23,15 @@ requirements:
     - setuptools
   run:
     - python
+    - certifi
 
 test:
   imports:
     - libcloud
 
+  commands:
+    - python certifi_test.py
```

Comment:
That seems reasonable. Though I think we need to add it to `files` under `test` so it moves to the testing directory.